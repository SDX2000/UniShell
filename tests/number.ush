1
123
echo 123