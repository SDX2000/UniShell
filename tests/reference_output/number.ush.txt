================
number.ush
================
1
123
123
